src/types/signable-token.ts

```diff
@@ -0,0 +1,48 @@
+import { SignableToken } from '../api';
+
+export interface ERC20Token {
+  type: 'ERC20';
+  tokenAddress: string;
+  decimals: number;
```

Comment:
decimals will most likely go as part of https://immutable.atlassian.net/browse/DX-1184

Reply:
Yes, it will be inferred from the token contract now for sending back to the API, but since undecimalised amounts are now used, `decimals` won't be required anymore.

Reply:
I've tested ERC20 decimals are not required in get signable endpoints for transfer/order/withdrawal endpoints. 
Removed decimals here: [064964e](https://github.com/immutable/imx-core-sdk/pull/149/commits/064964ea75ebb2336539c401fb697a48dbd14fea)